docs/reference/index-modules/merge.asciidoc

```diff
@@ -108,6 +108,7 @@ call for the index (try and aim to issue it on a low traffic time).
 [float]
 [[log-byte-size]]
 ==== log_byte_size
+deprecated[1.6.0, this policy will be removed in 2.0 in favor of the [tiered] policy]
```

Comment:
deprecated[1.6.0,This policy will be removed in 2.0 in favour of the <<tiered,tiered>> merge policy]